src/UI/Icons/Ellipsis.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const Ellipsis = ({ ...props }) => (
+    <svg
+        height='24'
+        viewBox='0 0 24 24'
+        width='24'
+        xmlns='http://www.w3.org/2000/svg'
+        {...props}
+    >
+        {/* eslint-disable-next-line max-len */}
```

Comment:
You can remove this eslint-disable as the next line has a shorter length than our max-len.

Reply:
I will leave this, because many of the svgs has a longer path, this way all of the other components can be used as a template for new SVGs :))